packages/devtools-reps/src/reps/tests/grip-array.js

```diff
@@ -412,22 +471,24 @@ describe("GripArray - Set", () => {
     component = renderRep({ mode: MODE.TINY });
     // FIXME: Tiny mode should render Set as "Set" and not "[…]".
```

Comment:
not in your patch, but here I think we can remove the comment. This has been fixed.